toolchain/sem_ir/formatter.cpp

```diff
@@ -1078,6 +1084,20 @@ class FormatterImpl {
     out_ << " " << buffer;
   }
 
+  // Format the metadata in File for `import Cpp`.
+  auto FormatInstRHS(ImportCppDecl /*inst*/) -> void {
+    out_ << " ";
+    OpenBrace();
+    for (ImportCpp import_cpp : sem_ir_->import_cpps().array_ref()) {
+      Indent();
+      out_ << "import Cpp \""
+           << FormatEscaped(
+                  sem_ir_->string_literal_values().Get(import_cpp.library_id))
+           << "\" = " << SemIR::LocId(import_cpp.node_id) << "\n";
```

Comment:
```suggestion
           << "\"\n";
```

As mentioned this morning, we're only putting locations in semir when it's required to disambiguate names. i.e., if there are two instructions get named `%Foo`, we might disambiguate the second as `%Foo.loc4_1` (specifically, I think the first still remains `%Foo` and later duplicates are get unique-ified)

Reply:
Done.